firefox_ui_harness/decorators.py

```diff
@@ -10,8 +10,11 @@
 
 def skip_if_e10s(target):
     def wrapper(self, *args, **kwargs):
-        if self.appinfo.browserTabsRemoteAutostart:
-            raise SkipTest("Skipping due to e10s")
+        try:
+            if self.appinfo.browserTabsRemoteAutostart:
```

Comment:
If this fails then there is something wrong in retrieving this property via the appinfo class. There should be no need to do any changes here.